### DISCUSSION TITLE
SoftEther  Support Kernels

### DISCUSSION THREAD

**Discussion by Sherry0808** [SoftEtherVPN/SoftEtherVPN#1910] [General]
Hi，

When will SoftEther support the 5.4 series of kernels?

**Comment by chipitsine** [comment]
it should be supported. what kind of issues do you have ?

**Comment by Sherry0808** [comment]
After installing Softether, I added the tap_28 interface of device A to vrf-14 and also added the tap_28 interface of device B to vrf-14. Then, I tried to connect using Softclient and access the 172.16.21.1 address of device B's tap_28 interface, but I found that it was not accessible. Our current kernel version is 5.4, so I would like to know if Softether supports the 5.4 kernel. I noticed on the official website that it explicitly states support for 2.x, 3.x, and 4.x kernels, so I want to confirm.

**Reply by chipitsine** [reply]
I confirm that kernel 5.4 is supported.

**Reply by chipitsine** [reply]
kernel version is one of numerous factors.

were you able to run on previous kernels ?
what are your error messages ? do processes run ? do they listen on network ?
